server/charts/historian/templates/historian-deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         options:
         - name: single-request-reopen
       imagePullSecrets:
-      - name: regsecret
+      - name: fluidcrregsecret
```

Comment:
Why?

Reply:
I created a new secret as fluidcrregsecret for the fluidcr.
The old regsecret remains there and keeps the same definition to pull images from prague.

Reply:
I would suggest to flip it. Update regsecret to point to new fluid registry and then create a new secret "legacyRegSecret" that points to Prague. Then you don't need to change these files.

Reply:
Otherwise, people using these deployment files will need to update their cluster.

Reply:
Got it. Changed it back. :)